cmd/build/test.go

```diff
@@ -143,15 +143,19 @@ func (t Test) Integration(ctx context.Context, self run.DependencyIDer, filter s
 		env["PKO_TEST_LATEST_BOOTSTRAP_JOB"] = url
 	}
 
+	coverPkg := dotFilepathJoin("...") + "," +
+		dotFilepathJoin("apis", "...") + "," +
+		dotFilepathJoin("pkg", "...")
```

Comment:
Not a file path, only use forward slashes

Reply:
Fixed